moped-editor/src/views/projects/ProjectSummary.js

```diff
@@ -0,0 +1,256 @@
+import React from "react";
+import { useQuery } from "@apollo/react-hooks";
+import { useNavigate } from "react-router-dom";
+import { gql } from "apollo-boost";
+import { makeStyles } from "@material-ui/core/styles";
+import Page from "src/components/Page";
+import PropTypes from "prop-types";
+import {
+  Button,
+  Box,
+  Container,
+  Card,
+  CardHeader,
+  CardContent,
+  Divider,
+  CircularProgress,
+  AppBar,
+  Tab,
+  Tabs,
+  Typography,
+  Grid,
+} from "@material-ui/core";
+
+const ProjectSummary = () => {
+  const useStyles = makeStyles(theme => ({
+    root: {
+      flexGrow: 1,
+      backgroundColor: theme.palette.background.paper,
+    },
+    cardWrapper: {
+      marginTop: theme.spacing(3),
+    },
+    buttons: {
+      display: "flex",
+      justifyContent: "flex-end",
+    },
+    button: {
+      marginTop: theme.spacing(3),
+      marginLeft: theme.spacing(1),
+    },
+  }));
+  const classes = useStyles();
+
+  const navigate = useNavigate();
+
+  const handleClick = () => {
+    navigate("/moped/dashboard");
+  };
```

Comment:
Maybe this should redirect back to the projects list

Reply:
sounds like a plan, I'll change it tonight.